cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnTwillRunnerService.java

```diff
@@ -322,7 +321,6 @@ public synchronized TwillController lookup(String applicationName, final RunId r
   }
 
   private void startUp() throws Exception {
-    yarnAppClient.startAndWait();
```

Comment:
Yes it is not required. YarnAppClient is not a service anymore in Twill 0.8.0. @anwar6953 Can you confirm?